fab/services/templates/supervisor_formplayer_spring.conf

```diff
@@ -1,6 +1,5 @@
 [program:{{ project }}-{{ environment }}-formsplayer-spring]
-command=java -jar -Xmx{{ formplayer_memory }} -Xss1024k {{ code_current}}/submodules/formplayer/build/libs/formplayer.jar --spring.config.name={{ environment }}
-user={{ sudo_user }}
```

Comment:
i see you also got rid of this line specifying the user, is that not needed anymore?